config/initializers/secret_token.rb

```diff
@@ -10,8 +10,8 @@
 # Make sure your secret_key_base is kept private
 # if you're sharing your code publicly.
 # Twinenyc::Application.config.secret_key_base = '5901998bc34d7504bb36b9aa578c6ae54a10e728b18fe3afd8f1114ca261c37ba08106d8c96d8b8323b9227c9c67fb029904d7419a64e9616b1a7b8bef9153d2'
-if Rails.env.match /development|test/
-  Twinenyc::Application.config.secret_key_base = 'x' * 128
-else
-  Twinenyc::Application.config.secret_key_base = ENV["SECRET_KEY_BASE"]
-end
\ No newline at end of file
+
+secure = !Rails.env.match(/development|test/)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.